### PR TITLE
Standalone node on PowerVS

### PR DESCRIPTION
**Pull request by sudeeshjohn**
This is a modified code of PR  https://github.com/ocp-power-automation/infra/pull/7

### CODE REVIEW

File: infra-node/LICENCE.txt
Comment (mkumatag) [inline infra-node/LICENCE.txt:1]: not needed as parent dir already got a license file

File: infra-node/data/.gitignore
Comment (mkumatag) [inline infra-node/data/.gitignore:1]: this can be removed

File: infra-node/infra-node.tf
Comment (yussufsh) [inline infra-node/infra-node.tf:15]: Prefix is not getting added if vm_id is "".

File: infra-node/infra-node.tf
Comment (yussufsh) [inline infra-node/infra-node.tf:42]: false here?

File: infra-node/outputs.tf
Comment (yussufsh) [inline infra-node/outputs.tf:35]: newline

File: infra-node/var.tfvars
Comment (yussufsh) [inline infra-node/var.tfvars:21]: vm_id and prefix can be added here.

File: infra-node/infra-node.tf
Comment (sudeeshjohn) [reply infra-node/infra-node.tf:42]: should work , updating

**Review by yussufsh — APPROVED** [review]

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/33#pullrequestreview-549025155" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/33#issuecomment-742479028" title="Approved">mkumatag</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/33#" title="Author self-approved">sudeeshjohn</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/33#pullrequestreview-549013423" title="Approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Finfra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)~~ [bpradipt,mkumatag,sudeeshjohn]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
